sbin/copyMinified.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
```

Comment:
Is this script portable so it will work on windows?

Reply:
Yes.

Reply:
ok